contributing/github-guide.md

```diff
@@ -162,4 +148,6 @@ If it says your repository is **ahead** it means you have commits that haven't b
 
 ![Sync](./images/update-branch.PNG)
 
-We use what are called "squash commits" to keep the main repository clean. All it means to you is that you should sync before starting each portion of new work.
\ No newline at end of file
+We use what are called "squash commits" to keep the main repository clean. All it means to you is that you should sync before starting each portion of new work.
+
+If your brownbook repository somehow gets to a state where dsiplays as both some number of commits **ahead and behind**, let Josh know and he can help you resolve it.
```

Comment:
Displays typo, might be another word missing as well

Reply:
Fixed (and fixed some other stuff as well!)